templates/docker-compose.template

```diff
@@ -48,41 +52,32 @@ services:
     restart: always
     depends_on:
       - helium-miner
-    expose:
-      - "1681"
+    network_mode: host
```

Comment:
why do we need to move everything to privileged and host networking?

Reply:
We forced to have privileged mode due to lack of actual devices. Like if you have defined a device but that device doesn't exist, that container never starts. We've introduced privileged mode for this.

Host networking is coming from gateway-rs as it was designed for not containerized environments. (@pritamghanghas knows this topic better than me)

Reply:
@MuratUrsavas we had it working before with non-host networking so I don't see how that can be true.

Re the devices - I don't understand your meaning? There is no need for privileged.